<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addCar</class>
 <widget class="QDialog" name="addCar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>980</width>
    <height>594</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="BrandlineEdit">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>180</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
border:2px solid grey;
color:white;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="BrandLabel">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>170</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lucida Fax</family>
     <pointsize>14</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 255, 255)</string>
   </property>
   <property name="text">
    <string>Brand:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="CarnolineEdit">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>120</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
border:2px solid grey;
color:white;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="CarnoLabel">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>110</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lucida Fax</family>
     <pointsize>14</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 255, 255)</string>
   </property>
   <property name="text">
    <string>Car no:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="AddCarbtn">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>500</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 255, 0);
border-radius:5px;</string>
   </property>
   <property name="text">
    <string>AddCar</string>
   </property>
  </widget>
  <widget class="QLabel" name="AddcarTitle">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>20</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lucida Fax</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 255, 255)</string>
   </property>
   <property name="text">
    <string>Add Car</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exitbtn">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>500</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(175, 175, 175);
border-radius:5px;</string>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </widget>
  <widget class="QLabel" name="carTypeLabel">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>220</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lucida Fax</family>
     <pointsize>14</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 255, 255)</string>
   </property>
   <property name="text">
    <string>CarType:</string>
   </property>
  </widget>
  <widget class="QListWidget" name="CarTypelistWidget">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>230</y>
     <width>151</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
border:2px solid grey;
color:white;</string>
   </property>
  </widget>
  <widget class="QLabel" name="FareLabel">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>320</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lucida Fax</family>
     <pointsize>14</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 255, 255)</string>
   </property>
   <property name="text">
    <string>Fare:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="fareLineEdit">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>340</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
border:2px solid grey;
color:white;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>-170</x>
     <y>0</y>
     <width>1151</width>
     <height>621</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>image/mahindra-scorpio.jpg</pixmap>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>BrandlineEdit</zorder>
  <zorder>BrandLabel</zorder>
  <zorder>CarnolineEdit</zorder>
  <zorder>CarnoLabel</zorder>
  <zorder>AddCarbtn</zorder>
  <zorder>AddcarTitle</zorder>
  <zorder>exitbtn</zorder>
  <zorder>carTypeLabel</zorder>
  <zorder>CarTypelistWidget</zorder>
  <zorder>FareLabel</zorder>
  <zorder>fareLineEdit</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
